.jenkins/pytorch/test.sh

```diff
@@ -413,4 +413,10 @@ else
     time python -mcoverage xml
     popd
   fi
+  if [[ "$BUILD_ENVIRONMENT" = *profile* ]]; then
+    pushd build
+    echo "Generating lcov coverage report for C++ sources"
```

Comment:
nit: can you also change the above message too?
```
echo "Generating lcov coverage report for Python sources"
```

Reply:
@walterddr why do you want to replace C++ with Python in this sentence? In my mind, phrase "Python source files" are `.py` files and `lcov` is pretty useless for those.
How about "Generating C/C++ code coverage report using lcov"?

Reply:
oh I meant line 412 --> somehow github doesn't allow me to comment on that line ...